src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -49,6 +50,10 @@
     public static final String[] CORRESPONDENT_EVENTS = {"CORRESPONDENT_DELETED", "CORRESPONDENT_CREATED", "CORRESPONDENT_UPDATED"};
     public static final String[] EXTENSION_EVENTS = {"EXTENSION_APPLIED"};
     public static final String[] ALLOCATION_EVENTS = {"STAGE_ALLOCATED_TO_TEAM", "STAGE_CREATED", "STAGE_RECREATED", "STAGE_COMPLETED", "STAGE_ALLOCATED_TO_USER", "STAGE_UNALLOCATED_FROM_USER"};
+    private static class StreamBrokenException extends RuntimeException {
```

Comment:
I'd prefer for this to be in it's own class, but not a deal breaker.

Reply:
Moved to external file